src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -41,7 +41,9 @@ class IngestUploadWdg(BaseRefreshWdg):
         'ingest_data_view': 'Specify a ingest data view, defaults to edit',
         'extra_data': 'Extra data (JSON) to be added to created sobjects',
         'oncomplete_script_path': 'Script to be run on a finished ingest',
-        'update_mode': 'Takes values "true" or "false".  When true, uploaded files will update existing file iff exactly one file exists already with the same name.'
+        'update_mode': 'Takes values "true" or "false".  When true, uploaded files will update existing file iff exactly one file exists already with the same name.',
```

Comment:
a little picky but we use the bare infinitive in these instructions: Take, 
                                                                                                   Set or remove